src/openforms/api/fields.py

```diff
@@ -1,7 +1,7 @@
-from rest_framework import relations, serializers
+from rest_framework import relations
 
 
-class PrimaryKeyRelatedAsChoicesField(serializers.PrimaryKeyRelatedField):
+class PrimaryKeyRelatedAsChoicesField(relations.PrimaryKeyRelatedField):
```

Comment:
let's revert this:

![image](https://github.com/open-formulieren/open-forms/assets/5518550/21915611-017f-4564-be78-3b550ba34987)

https://www.django-rest-framework.org/api-guide/relations/